### PR TITLE
[CFM-938] -Upload attachments in Quickpost

### PR DESCRIPTION
**Pull request by ybaras**
[CFM-938](https://issuetracker.amplexor.com/jira/browse/CFM-938)

### CODE REVIEW

**Comment by ybaras** [comment]
@ordavidil New PR opened.

**Comment by ybaras** [comment]
capacity4more

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ?>
 <form name="entityForm"
       ng-submit="submitForm(data, selectedResource, 'quick_post')"
-      xmlns="http://www.w3.org/1999/html">
+      xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html">
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php]: @ybaras 
why did you change that one ?
it is now duplicated ...

**Comment by anvmn** [comment]
out

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php (hunk 1 above)
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php]: @ybaras, @ordavidil 
Any new with this?

File: project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module [hunk 2]
```diff
@@ -475,7 +475,7 @@ function _c4m_document_widget_document_file($file_id, $is_quick_post = FALSE) {
   // PURL options.
   $options_context = array(
     'purl' => array(
-      'disabled' => FALSE,
+      'enabled' => TRUE,
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module]: @anvmn, please verify this one with @ybaras

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module]: @ybaras 
I've tested this, and verified purl is properly created when we have `'enabled' => TRUE`.
Please approve and update your PR.

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php [hunk 3]
```diff
@@ -31,6 +31,14 @@ protected function checkEntityAccess($op, $entity_type, $entity) {
     if (!in_array($group_status, $allowed_groups) && !user_access('administer site configuration', $account)) {
       return FALSE;
     }
+
+    if ($resource_name == 'documents') {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php]: @ybaras 
with your PR these lines are not needed, right ?

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php (hunk 1 above)
Comment (ybaras) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php]: It looks like a typo... Revert it.

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php (hunk 3 above)
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php]: @anvmn 
please remove this one

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful/plugins/restful/C4mRestfulEntityBaseNode.class.php]: @ordavidil 
Removed

File: project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module (hunk 2 above)
Comment (ybaras) [inline project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module]: This change does not solve the problem in my case.
Tested by attaching a document while creating a new event in group nobelprize as alfrednobel (GO). 
I will collect the gid from the post data in the C4MOgSelectionHandler class.

Comment (ybaras) [inline project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module]: @anvmn This modification will cause an error in attaching files + adding to library. We should discuss.

**Comment by anvmn** [comment]
out

**Comment by ordavidil** [comment]
Already merged under #1432
